ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -2708,6 +2708,15 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-vsphere-ipi-remote-worker
+- as: vsphere-ipi-sdn-live-migration-ovn-f28
+  cron: 39 9 1,17 * *
+  steps:
+    cluster_profile: vsphere-2
+    env:
+      OVERRIDE_OPENSHIFT_SDN_DEPRECATION: "true"
+    test:
+    - chain: cucushift-sdn-live-migration-ovn
+    workflow: cucushift-installer-rehearse-vsphere-ipi-sdn
 - as: vsphere-ipi-sdn-migration-ovn-f14
```

Comment:
Guess we can remove this one if live migration ready for customer?

Reply:
@zhaozhanqi Agree, we can remove the offline migration workflow until this new live migration released in 4.16.

Reply:
can we remove this profile in 4.16 if so?

Reply:
@zhaozhanqi Done!

Reply:
@zhaozhanqi @liangxia Please let me know if you have any more comments. Hope it can be merged before my PTO starting from 5/9, thanks!

Reply:
thanks
/lgtm